tests/test.sh

```diff
@@ -21,7 +28,7 @@ function cleanup {
 }
 
 # Calls cleanup when the script exits
-trap cleanup EXIT
+#trap cleanup EXIT
```

Comment:
I'd recommend wrapping this in a conditional that only triggers the trap if $1 is not provided